api/grpc/disperser/v2/disperser_v2.pb.go

```diff
@@ -156,14 +166,21 @@ func (x *DisperseBlobRequest) GetBlobHeader() *v2.BlobHeader {
 	return nil
 }
 
+// A reply to a DisperseBlob request.
 type DisperseBlobReply struct {
 	state         protoimpl.MessageState
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
 
 	// The status of the blob associated with the blob key.
-	Result  BlobStatus `protobuf:"varint,1,opt,name=result,proto3,enum=disperser.v2.BlobStatus" json:"result,omitempty"`
-	BlobKey []byte     `protobuf:"bytes,2,opt,name=blob_key,json=blobKey,proto3" json:"blob_key,omitempty"`
+	Result BlobStatus `protobuf:"varint,1,opt,name=result,proto3,enum=disperser.v2.BlobStatus" json:"result,omitempty"`
+	// The unique 32 byte identifier for the blob.
+	//
+	// The blob_key is the keccak hash of the rlp serialization of the BlobHeader, as computed here:
+	// https://github.com/Layr-Labs/eigenda/blob/0f14d1c90b86d29c30ff7e92cbadf2762c47f402/core/v2/serialization.go#L30
+	// The blob_key must thus be unique for every request, even if the same blob is being disperser.
```

Comment:
is being disperser -> is being dispersed

Reply:
yes, fixed

TODO commit